test/org/sagebionetworks/bridge/play/controllers/ApplicationControllerMockTest.java

```diff
@@ -109,5 +126,37 @@ public void startSessionWorks() throws Exception {
         assertEquals("email", signIn.getEmail());
         assertEquals("token", signIn.getToken());
     }
+    
+    @Test
+    public void androidAppLinks() throws Exception {
+        study.getAndroidAppLinks().add(TestConstants.ANDROID_APP_LINK);
+        study.getAndroidAppLinks().add(TestConstants.ANDROID_APP_LINK_2);
```

Comment:
Is it worth creating a test with multiple studies (and multiple links per study) to make sure this correctly accumulates links across multiple studies? (For both this and the Apple version.)

Reply:
Yes, added tests for both.